extension/content.js

```diff
@@ -409,6 +409,13 @@ function fixSquashAndMergeTitle() {
 	});
 }
 
+function addTitleToEmojis() {
+	const emojis = $('.js-comment-body, #readme').find('g-emoji');
```

Comment:
@hkdobrev shall I change it to affect all of the `g-emoji` then? No matter where on the page they are displayed?